src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -442,6 +443,12 @@ private function getErrorMessages(Exception $e): array
                 'The selected value belongs to another feature.',
                 'Admin.Notifications.Error'
             ),
+            SpecificPriceConstraintException::class => [
+                SpecificPriceConstraintException::DUPLICATE_PRIORITY => $this->trans(
+                    'Specific price priorities cannot duplicate',
```

Comment:
What does it mean exactly? Do you mean that the order of priority set for a particular product cannot be the same as the default order of priority?

Reply:
![Screenshot from 2022-01-10 16-47-39](https://user-images.githubusercontent.com/31609858/148785323-27489a8c-f0c9-484a-917e-bcecd54cd161.png)
e.g. error appears if you select it this way (currency 2 times). The ux will be improved and probably this error will not depend on merchant behavior anymore, but thats for other pr's.